### ISSUE TITLE
Consider accessibility

### ISSUE THREAD

**Issue by dteal** [ut-ras/ras_site_old#33]
The current HTML markup probably meets some kind of bare minimum accessibility standard, but the website should be run through some kind of accessibility-checking tool.
